ddl/partition.go

```diff
@@ -216,3 +216,11 @@ func onDropTablePartition(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	job.Args = []interface{}{partitionID}
 	return ver, nil
 }
+
+// TODO: We haven't checked subpartitions yet.
```

Comment:
This TODO will never be done, how about removing it?

Reply:
Agree.